### DISCUSSION TITLE
Fundamentos do C#: Do/While na Prática - Esc do editor de textos não funciona direito

### DISCUSSION THREAD

**Discussion by RomualdoM** [balta-io/forum#539] [Tira Dúvidas] [answered]
Boas tardes!
"Meu" código para o programa de edição de textos ficou praticamente idêntico ao do prof. Balta. Entretanto, a tecla "esc" só funciona se for antecedida imediatamente de um enter, sem nenhuma outra digitação. Uso o terminal _bash_ do Linux. Será que tem a ver com isso?
Obrigado!

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado
https://github.com/balta-io/forum/discussions/41
